lib/_stream_pump.js

```diff
@@ -0,0 +1,191 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2014 Mathias Buus
```

Comment:
On a semi-related note, I wish we could collect any/all vendored code under a single directory to better indicate that PRs/issues/etc. should not be opened in nodejs/node for those files.